file.go

```diff
@@ -166,20 +166,39 @@ func (f *File) UnmarshalJSON(p []byte) error {
 	return errors.New("json struct tag unmarshal is deprecated, use ach.FileFromJSON instead")
 }
 
+// ACHFileToJSON returns an ACH file in JSON format from ACH.File.
+func (f *File) ACHFileToJSON() ([]byte, error) {
```

Comment:
I don't see why this method is needed. Callers can just issue `json.Marshal(fileJSON)` themselves.

Reply:
That’s correct, but I wanted in the library.

Reply:
Could we name it `ToJSON` then?

Reply:
Yes, I think so, if that is preferable.

Reply:
Let's rename this to `ToJSON`. The current name is wordy.

```Go
file.ACHFileToJson()

// vs

file.ToJSON() 
```

Reply:
I’m going to use MarshalIndent in the caller